README.md

```diff
@@ -172,7 +172,7 @@ To build the server JAR, execute the following:
 $ ./sbt "project server" assembly
 ```
 
-which should result in `server/target/scala-2.10/usaceflood-server-assembly-SNAPSHOT.jar`.
+which should result in `server/target/scala-2.10/usaceflood-server-assembly-<version>-SNAPSHOT.jar`.
```

Comment:
Consider also updating the JAR name in other places, like https://github.com/azavea/usace-flood-geoprocessing/pull/27/files#diff-04c6e90faac2675aa89e2176d2eec7d8R193